bee-network/bee-autopeering/Cargo.toml

```diff
@@ -46,3 +47,6 @@ prost-build = { version = "0.8", default-features = false }
 [[example]]
 name = "node"
 
+[features]
+default = [ "rocksdb" ]
+in-memory = [ ]
```

Comment:
Shouldn't this section also explicitly announce the `sled` and `rocksdb` feature, and then make `default` point to `rocksdb` feature? I think that would be nicer from a UX pov.

Reply:
sadly this is illegal for now: https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies